modules/core/jvm/src/test/scala/coursier/core/ObjectSizeTests.scala

```diff
@@ -57,30 +57,34 @@ object ObjectSizeTests extends TestSuite {
       }
     }
 
-    test("Dependency memoised_cache should hold objects until they can be GCd") {
+    test("Dependency instanceCache should hold objects until they can be GCd") {
       test("should be the different for different dependency") {
         implicit val retryMax = RetryMax(5.seconds)
         implicit val retryInterval = RetryInterval(200.millis)
-        Dependency.memoised_cache.clear();
+        def cacheSize(): Int = {
+          Dependency.instanceCache.purgeStaleEntries()
```

Comment:
Why you need to call purgeStaleEntries?

Reply:
Because unlike `get` for example, `size` still includes stale entries (whose keys and / or values where garbage-collected).